### ISSUE TITLE
This does not properly extract Surviving Mars files

### ISSUE THREAD

**Issue by thecookiemaker** [nickelc/hpk#8]
when extracting the Surviving Mars .hpk files it appears to work correctly, but if you open any of the resulting files they aren't valid. For example if you open a .Lua file the text is unreadable. If you attempt to open a .tga it is not recognized as a valid picture file.

I downloaded this and ran it fresh today just to verify.

**Comment by nickelc** [comment]
Thank you for testing.

I've now tested more files of the gog version and could not reproduce the problem.
Tested Files:
**Local/English.hpk:** All extracted csv files look fine.
**Packs/CommonAssets.hpk:** i could open all tested tga files with gimp without any issue.
**Packs/Music.hpk:** no playback issue with the opus files.
**Packs/Lua.hpk:** The lua files are compiled and contain only the bytecode. I checked some files with a hexeditor and they have a valid magic code `1B 4C 75 61 53 00` for bytecode of Lua 5.3.
